src/_layouts/generic.njk

```diff
@@ -16,16 +16,17 @@
   {{ govukHeader({
     productName: "Brand Guidelines"
   }) }}
-  {{ govukServiceNavigation({
-    navigation: [
-      { href: "#", text: "Graphic device" },
-      { href: "#", text: "Logo" },
-      { href: "#", text: "Colour" },
-      { href: "#", text: "Typography" },
-      { href: "#", text: "Visualising data" },
-      { href: "#", text: "Brand in use" }
-    ]
-  }) }}
+
+  {% for ancestor in ancestors %}
+      {{ govukServiceNavigation({
+        navigation: ancestor.data.navigationItems
+      }) }}
+  {% endfor %}
+  {% if navigationItems | length %}
+    {{ govukServiceNavigation({
```

Comment:
This one needs a different `navigationId` set so that the two menu's JS doesn't conflict. Should probably have a different `menuButtonText` as well.

Reply:
Oooh, that's a good point, I'll work something out based on the page whose navigation it shows. Navigation UI is all throwaway, but I should at least make it work properly on mobile.